app/uk/gov/hmrc/partnershipidentificationfrontend/api/controllers/JourneyConfigJsonReader.scala

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
If this class is currently not being used I would remove it from the branch and keep a local copy in case we choose to use this approach in the future.

Reply:
removed